lms/templates/problem_notifications.html

```diff
@@ -4,9 +4,10 @@
 from openedx.core.djangolib.markup import HTML
 %>
 
-<div class="notification ${notification_type}" tabindex="-1">
+<div class="notification ${notification_type} ${'notification-hint' if hint_notification else 'notification-response'}" tabindex="-1">
     <span class="icon fa fa-${notification_icon}" aria-hidden="true"></span>
-    <span class="notification-message">${notification_message}</span>
+    <span class="notification-message" aria-describedby="${notification_message}" aria-live="${notification_message}">${notification_message}
```

Comment:
The value of aria-describedby has to be an ID (or list of IDs). And the ID must be unique if there are multiple problems on the same page.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute